murenn/dtcwt/transform1d.py

```diff
@@ -16,7 +16,7 @@ def __init__(
         J=8,
         skip_hps=False,
         include_scale=False,
-        alternate_gh=True,
+        alternate_gh=False,
```

Comment:
i'm still curious to know if this breaking change is still useful

Reply:
i think this is still useful